src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1037,6 +1037,74 @@
         az acr token update -n MyToken -r MyRegistry --scope-map MyNewScopeMap
 """
 
+helps['acr agentpool'] = """
+type: group
+short-summary: Manage private Tasks agent pools with Azure Container Registries.
+"""
+
+helps['acr agentpool create'] = """
+type: command
+short-summary: Create an agent pool for an Azure Container Registry.
+examples:
+  - name: Create the agent pool 'MyAgentName' associated with the registry 'MyRegistry'.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry
+  - name: Create the agent pool 'MyAgentName' with n agent count.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry --count n
+  - name: Create the agent pool 'MyAgentName' associated with the registry 'MyRegistry' in VNET 'MyVnetId'.
+    text: >
+        az acr agentpool create -n MyAgentName -r MyRegistry --vnet-id MyVnetId
+"""
+
+helps['acr agentpool update'] = """
+type: command
+short-summary: Update an agent pool for an Azure Container Registry.
+examples:
+  - name: Update the agent pool 'MyAgentName' count
+    text: >
+        az acr agentpool update -n MyAgentName -r MyRegistry --tag NewCount
```

Comment:
confused. What is a tag?

Reply:
```
examples:
  - name: Scale up my agentpool to 3 agents.
    text: >
        az acr agentpool update -n MyAgentName -r MyRegistry --count 3
  - name: Scale down my agentpool to 0 agents.
    text: >
        az acr agentpool update -n MyAgentName -r MyRegistry --count 0

```

Reply:
@huanwu it reminds me since agent pool is tracked resource, do we need to do anything in rp to support tags? same question for task run.

Reply:
I double checked the swagger and seems the swagger will always generate the arm resource tags for the UpdateParameters including task and taskrun. What kind of scenario will we use the tags?

Reply:
I was confused as well but I saw it in both the data model for update and create. So I set the variable I can hide for now with the deprecate functionally to block it